### DISCUSSION TITLE
Insert list of object datatype in milvus

### DISCUSSION THREAD

**Discussion by rahuldhagote** [milvus-io/milvus#21715] [Q&A and General discussion] [answered]
I have data in my columns 
0     [jnm1851smss]
1     [je510bw, je520wc, je520bw, je510wc]
2     [zsc2201jss, zsc2202jss]
3     [sca2000baa]

I want to upload this data to milvus as it is, so at the time of hybrid search I can give any one element from list to filter, how can I do this? I dont want to convert it in string

currently I am getting this error
<DataTypeNotSupportException: (code=1, message=Field dtype must be of DataType)>

**Comment by xiaofan-luan** [comment, marked as answer]
Milvus doesn't support list/set yet.
It's in the milestone of 2.4 Major release (Some time in May), so before that you still need other database to store complex datatypes.
